app/services/github/contributions/pull_request_collector.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Github
+  module Contributions
+    class PullRequestCollector
+      LINK_INDEX = {
+        org: 3,
+        repo: 4,
+        pr: 6
+      }
```

Comment:
I think this one can be a one-liner
```suggestion
      LINK_INDEX = { org: 3, repo: 4, pr: 6 }
```

Reply:
Done!